Updated about RHDH (#1157)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/assembly-about-rhdh.adoc

:context: about-rhdh

> {product} ({product-very-short}) is an enterprise-grade internal developer portal (IDP) that simplifies and accelerates software delivery. It provides a customizable web-based interface that centralizes access to key development resources, including source code repositories, CI and CD pipelines, APIs, documentation, and runtime environments.

Reviewer: We might have intentionally used language like “is designed to xyz” because directly claiming that it does xyz might be a promise that comes with legal implications. Just something to consider.

> {product} ({product-very-short}) is an enterprise-grade internal developer portal (IDP) that simplifies and accelerates software delivery. It provides a customizable web-based interface that centralizes access to key development resources—including source code repositories, CI/CD pipelines, APIs, documentation, and runtime environments. The platform supports cloud-native environments such as Kubernetes, {ocp-brand-name}, and hybrid infrastructure.

Reviewer 2: https://github.com/redhat-documentation/vale-at-red-hat/pull/938 to address the following warning:

[vale] reported by reviewdog 🐶
[RedHat.Slash] Use either 'or' or 'and' in 'CI/CD'

> {product} ({product-very-short}) is an enterprise-grade internal developer portal (IDP) that simplifies and accelerates software delivery. It provides a customizable web-based interface that centralizes access to key development resources, including source code repositories, CI and CD pipelines, APIs, documentation, and runtime environments. The platform supports cloud-native environments such as Kubernetes, {ocp-brand-name}, and hybrid infrastructure.

Reviewer 3: ```suggestion
{product} ({product-very-short}) is an enterprise-grade internal developer portal (IDP) that simplifies and accelerates software delivery. It provides a customizable web-based interface that centralizes access to key development resources, including source code repositories, CI and CD pipelines, APIs, documentation, and runtime environments.
```

> The result is faster delivery, reduces tool sprawl, and standardized practices across teams.

Reviewer 3: ```suggestion
RHDH is designed for cloud-native environments, including Kubernetes, {ocp-brand-name}, and hybrid infrastructure. By consolidating tools and standardizing development workflows, it helps teams deliver software faster and with greater consistency.
```

> RHDH is designed for cloud-native environments, including Kubernetes, {ocp-brand-name}, and hybrid infrastructure. By consolidating tools and standardizing development workflows, it helps teams deliver software faster and with greater consistency.

Reviewer: Replace product names with attributes

> RHDH is designed for cloud-native environments, including Kubernetes, {ocp-brand-name}, and hybrid infrastructure. By consolidating tools and standardizing development workflows, it helps teams deliver software faster and with greater consistency.

Reviewer: ```suggestion
RHDH is designed for cloud-native environments, including supported Kubernetes platforms, {ocp-brand-name}, and hybrid infrastructure. By consolidating tools and standardizing development workflows, it helps teams deliver software faster and with greater consistency.
```

Just a suggestion but probably fine as-is

> Designed for enterprise-scale software teams, {product-very-short} offers a fully supported, turnkey developer portal experience. It features simplified installation, dynamic plugin management, and seamless integration with existing tools. With enterprise grade security, role-based access control, and 24x7 support, teams can focus on building software instead of maintaining internal infrastructure.

Reviewer 3: ```suggestion
Designed for enterprise-scale software teams, {product-very-short} offers a fully supported, turnkey developer portal experience. Key features include simplified installation, dynamic plugin management, and seamless integration with existing tools. With enterprise-grade security, role-based access control, and 24x7 support, teams can focus on building and delivering software rather than maintaining internal infrastructure.

```

> Designed for enterprise-scale software teams, {product-very-short} offers a fully supported, turnkey developer portal experience. Key features include simplified installation, dynamic plugin management, and seamless integration with existing tools. With enterprise-grade security, role-based access control, and 24x7 support, teams can focus on building and delivering software rather than maintaining internal infrastructure.

Reviewer: It's cool to list features, but we also want to help our users understand how those features help them achieve their goals. User-centric content > product-focused content. :)
